frontend/src/components/AddMember/index.js

```diff
@@ -0,0 +1,150 @@
+
+  const options2 = [
+    { key: "1", value: "Auto izolare" },
+    { key: "2", value: "Carantină la domiciliu" },
+    { key: "3", value: "Carantină specializată" },
+    { key: "4", value: "Niciuna" }
+  ];
+
+  const genderOptions = [
+    { key: "", text: "" },
+    { key: "1", text: "Feminin" },
+    { key: "2", text: "Masculin" }
+  ];
+
+  var ageOptions = [...Array(100).keys()].map(index => ({
+    text: index,
+    value: index
+  }));
+  ageOptions[0].text = "";
+
+  return (
+    //TODO: remove common components when added to BasePage
+    <BasePage>
+      <Hero
+        title="Ce pași ai de urmat"
+        subtitle="Pentru a te putea ajuta iata ce ai la dispozitie in contul tau:"
+        useFallbackIcon={true}
+      />
+      <StepsBar />
+
+      <div className="columns">
+        <div className="column is-8">
+          <form>
+            <Hero
+              title="Creează cont pentru un membru al familiei"
+              subtitle="Înregistrează un membru al familiei care nu poate accesa platforma și ajută-l să își facă evaluarea zilnică a simptomelor COVID-19."
+            />
+            <Input label="Nume și prenume" type="text" required="true" />
+            <div className="columns">
+              <div className="column is-6">
+                <Select
+                  label="Legătura familiala"
+                  options={relationshipOptions}
+                />
+              </div>
+            </div>
+            <div className="columns">
+              <div className="column is-3">
+                <Select label="Vârstă" options={ageOptions} />
+              </div>
+              <div className="column is-3">
+                <Select
+                  label="Gen"
+                  options={genderOptions}
+                  selectProps={{ className: "is-extended" }}
+                />
+              </div>
+            </div>
+            <Label text="Are condiții de sănătate preexistente?" />
+            <p className="subtitle is-6">
+              Spune-ne dacă suferă de anumte boli cronice, diabet,
+              hipertensiune, etc
+            </p>
+            <Input label="" name="" />
+            <Label text="Are anumite dizabilități?" />
+            <p className="subtitle is-6">
+              Spune-ne dacă sferă de anumite dizabilitati Ex locomotorii,
+              mentale, de vorbire etc.
+            </p>
+            <RadioList label="" type="horizontal" options={options1} />
+            <RadioList
+              label="În ultima perioadă a fost în:"
+              type="horizontal"
+              options={options2}
+            />
+            <div className="is-pulled-right">
+              <Button
+                type="primary"
+                size="large"
+                disabled="true"
+                inputType="submit"
+              >
+                Adaugă
+              </Button>
+            </div>
+          </form>
+        </div>
+        <aside className="column is-4">
```

Comment:
Could you please move the `aside` in `BasePage` component, as the sidebar is shown on all pages

Reply:
Please also move the layout elements, like `.columns` and `.column .is-8`

Reply:
The `StepsBar` should also be moved to the `BasePage` in this case. This might affect some pages that don't have the `StepsBar` (e.g. Creare cont). I can add a flag on `BasePage` to hide the `StepsBar` if true, and by default to show it.

Reply:
Good point! In this case, could you please create a component, say `SidebarLayout`, that wraps the columns and the sidebar, so we can re-use that across all pages ?

Reply:
of course.